setup.py

```diff
@@ -19,13 +21,10 @@
     "posthog",
 ]
 
-DEV = [
-    "pytest",
-    "flake8",
-    "twine",
-    "invoke",
-    "pkgmt",
-]
+DEV = ["pytest", "flake8", "twine", "invoke", "pkgmt"]
+
+if platform.system() == "Windows":
+    DEV.append("pywin32")
```

Comment:
this won't work for built distributions (.whl) ,we can  add add a  new section in extras_require to account for this: https://stackoverflow.com/a/6470373/709975

Reply:
yeah, that sounds good